src/apps/common/submodules/numbers/numbers.js

```diff
@@ -1085,6 +1074,9 @@ define(function(require) {
 				.empty()
 				.append(template);
 
+			var arrayNumbersSpare = dataNumbers.listAccounts.length ? dataNumbers.listAccounts[0].spareNumbers : [];
+			self.numbersDisplayFeaturesMenu(arrayNumbersSpare, template);
+
 			callback && callback();
```

Comment:
To follow previous practices, this could be changed:
```suggestion
			args.hasOwnProperty('callback') && args.callback();
```